manywheel/build_docker.sh

```diff
@@ -57,9 +57,10 @@ DOCKER_IMAGE=${DOCKER_REGISTRY}/pytorch/manylinux-builder:${DOCKER_TAG}
     docker tag ${DOCKER_IMAGE} ${LEGACY_DOCKER_IMAGE}
 )
 
-if [[ -n ${WITH_PUSH} ]]; then
+if [[ "${WITH_PUSH}" == true ]]; then
```

Comment:
What's the reasoning behind this change? Wouldn't it also just work without needing to change the actual equals value here?

Reply:
It's cuz I set it to ${{ github.event_name == 'push' }}, which I think would be 'false' if we're not pushing, which would pass the `-n`, no?

Reply:
Oh that's true, we should probably document this in the README.md

Perhaps we can also make this change for the rest of the docker build scripts as well?